src/lib.rs

```diff
@@ -122,21 +115,25 @@ const RECOMMENDED_SECTION_SIZE: usize = 60;
 
 /// Number of elders per section.
 const ELDER_SIZE: usize = 7;
+/// Strict majority of Elders
+const ELDER_MAJORITY: usize = 4;
 
+/// Number of votes required to agree
+/// with a strict majority (i.e. > 50%)
+pub(crate) fn majority(num_possible_voters: usize) -> usize {
+    1 + (num_possible_voters / 2)
+}
 #[cfg(test)]
 mod tests {
-    use super::{MAJORITY_DENOMINATOR, MAJORITY_NUMERATOR};
+    use crate::{ELDER_MAJORITY, ELDER_SIZE};
+
+    use super::majority;
 
     #[test]
-    #[allow(clippy::assertions_on_constants)]
-    fn quorum_check() {
-        assert!(
-            MAJORITY_NUMERATOR < MAJORITY_DENOMINATOR,
-            "Majority impossible to achieve"
-        );
-        assert!(
-            MAJORITY_NUMERATOR * 2 >= MAJORITY_DENOMINATOR,
-            "Majority does not guarantee agreement"
-        );
+    fn strict_majority() {
+        assert_eq!(majority(4), majority(5));
+        assert_eq!(majority(6), majority(7));
+        assert_eq!(majority(7), 4);
```

Comment:
Maybe worth adding more asserts here:
```rust
assert_eq!(majority(0), 1) // pathological case
assert_eq!(majority(1), 1)
assert_eq!(majority(2), 2)
assert_eq!(majority(3), 2)
assert_eq!(majority(4), 3)
assert_eq!(majority(5), 3)
assert_eq!(majority(6), 4)
```
Possibly even proptest this?

Reply:
I have a pr to add proptest here. I can put that up next (branced off here)